### PR TITLE
P1 typography

### PR DESCRIPTION
**Pull request by Omkar-Ghate**
For the typography component. Please review the code and design.

### CODE REVIEW

File: pages/components/typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  font-weight: 700;
+}
+.text-light-weight {
+  font-weight: 300;
+}
+.text-regular-weight {
+  font-weight: 500;
+}
+.text-small-size {
+  font-size: 14px;
```
Comment (zoef-yt) [inline pages/components/typography/typography.css]: use rem

File: pages/components/css/utils.css [hunk 2]
```diff
@@ -153,7 +153,7 @@ p, .p{
   font-size: 0.875rem;
```
Comment (zoef-yt) [inline pages/components/css/utils.css:153]: us px

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: pages/components/typography/typography.css (hunk 1 above)
Comment (Omkar-Ghate) [reply pages/components/typography/typography.css]: it's below 1 rem right?

File: pages/components/css/utils.css [hunk 3]
```diff
@@ -253,6 +254,66 @@ p, .p{
...
+.list-inline-item:not(:last-of-type){
+  margin-right: 1.5rem;
+}
+.styled-list{
+  margin-bottom: 1rem;
+  font-family: var(--DOC-DESCRIPTION);
+}
+.styled-list li:not(:last-child){
+  padding-bottom: 0.625rem;
+}
```
Comment (zoyatanweer) [inline pages/components/css/utils.css:274]: you can use px here

File: pages/components/typography/typography.css [hunk 4]
```diff
@@ -0,0 +1,157 @@
+.main-heading {
+  margin-top: 100px;
```
Comment (zoyatanweer) [inline pages/components/typography/typography.css:2]: use rem

**Review by zoyatanweer — APPROVED** [review]

**Pull request merged**